doc/jenkins-guide.md

```diff
@@ -160,6 +161,61 @@ then it is probably appropriate to clean out part of the worker's
 workspace (where Jenkins jobs are performed). To clean out part of the
 workspace, run `rm -rf ~/build/workspace/node-test-commit*`.
 
+#### IBM platforms
```

Comment:
I'd suggest just titling this section as `AIX` and then we'd just list further platforms under their own headings.